index.html

```diff
@@ -1685,6 +1685,83 @@ <h3>Protocol Binding</h3>
         MUST implement this protocol binding.
         </span>
       </p>
+
+      <p>
+        The examples provided throughout this section describe how a Consumer
+        would communicate with a Web Thing which produces the following Thing
+        Description:
+      </p>
+
+      <pre class="example">
+        {
+          "@context": "https://www.w3.org/2019/wot/td/v1",
+          "id": "https://mywebthingserver.com/things/lamp",
+          "profile": "https://www.w3.org/2022/wot/profile/http-baseline/v1",
+          "base": "https://mywebthingserver.com/things/lamp/",
+          "title": "My Lamp",
+          "description": "A web connected lamp",
+          "securityDefinitions": {
+            "oauth2": {
+              "scheme": "oauth2",
+              "flow": "code",
+              "authorization": "https://mywebthingserver.com/oauth/authorize",
+              "token": "https://mywebthingserver.com/oauth/token"
+            }
+          },
+          "security": "oauth2",
+          "properties": {
+            "on": {
+              "type": "boolean",
+              "title": "On/Off",
+              "description": "Whether the lamp is turned on",
+              "forms": [{"href": "./properties/on"}]
+            },
+            "level" : {
+              "type": "integer",
+              "title": "Brightness",
+              "description": "The level of light from 0-100",
+              "unit": "percent",
+              "minimum" : 0,
+              "maximum" : 100,
+              "forms": [{"href": "./properties/level"}]
+            }
+          },
+          "actions": {
+            "fade": {
+              "title": "Fade",
+              "description": "Fade the lamp to a given level",
+              "input": {
+                "type": "object",
+                "properties": {
+                  "level": {
+                    "type": "integer",
+                    "minimum": 0,
+                    "maximum": 100,
+                    "unit": "percent"
+                  },
+                  "duration": {
+                    "type": "integer",
+                    "minimum": 0,
+                    "unit": "milliseconds"
+                  }
+                }
+              },
+              "forms": [{"href": "./actions/fade"}]
```

Comment:
why "./" prefix?

Reply:
I added the prefix for the top level "properties", "actions" and "events" endpoints to make it clearer that the `href` member is a path, then the rest are just there for consistency. I've removed the prefixes now though since apparently they are confusing.